gtfs_realtime_translators/translators/septa.py

```diff
@@ -3,21 +3,19 @@
 
 import pendulum
 
-from gtfs_realtime_translators.factories import TripUpdate, FeedMessage
+from gtfs_realtime_translators.factories import TripUpdate, FeedMessage, RequiredFieldValidator
 
 
-class VtaGtfsRealtimeTranslator:
-    def __init__(self, stop_id=None):
-        if stop_id is None:
-            raise ValueError('stop_id is required.')
-
-        self.stop_id = stop_id
```

Comment:
We should continue to pass the stop_id as an argument in the translator 

example:
```
        "translator_custom_data": "{\"stop_id\": \"4773\"}",

```

Reply:
I guess using stop_id in constructor is required for those feed or provider where you do not get stop id as a part of feed. In case of vta and septa, we will get it. Why do you want to add it back? any specific reason? It should work without this constructor. Only thing is that we will have to update all fields.